ttnn/cpp/ttnn/operations/conv/conv2d/device/kernels/conv_bmm_tilize_col_major_out_blocks.cpp

```diff
@@ -46,6 +46,44 @@ inline void tilize_in(
     tilize_uninit(in_cb_id);
 } // tilize_in()
 
+#ifdef USE_MAX_CORES
+template <uint32_t out_subblock_w, uint32_t out_block_w>
+inline void reblock_and_untilize_rows(
+    uint32_t num_out_subblocks_in_col,
+    uint32_t out_subblock_num_tiles,
+    uint32_t out_subblock_h,
+    uint32_t output_rows_h,
+    uint32_t interm_cb_id,
+    uint32_t out_cb_id) {
+    uint32_t num_tiles_in_row_of_subblocks = mulsi3(out_subblock_num_tiles, num_out_subblocks_in_col);
+    cb_wait_front(interm_cb_id, num_tiles_in_row_of_subblocks);
+
+    uint32_t within_block_index = 0;
+    for (uint32_t h = 0; h < out_subblock_h; h++) {
+        uint32_t block_offset = 0;
+        uint32_t test_out_block_w = output_rows_h <= 32 ? output_rows_h : 32;
```

Comment:
test_out_block_w, should we renama this one?
Feels like we could merge this f() and function below most of the code is the same

Reply:
Yeah only difference in both functions are the current one takes #f rows as a parameter fuction pack_untilize_dst. I was also thinking about merging these 2 fuctions although I was concern about overuse of the macro.

Reply:
can you get away w/o using macro and just use if constexpr ...